src/id128/id128.c

```diff
@@ -212,6 +217,13 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case 'p':
                         arg_mode = ID128_PRINT_PRETTY;
+                        arg_value = false;
+                        break;
+
+                case 'P':
+                        arg_value = true;
+                        if (arg_mode == ID128_PRINT_PRETTY)
```

Comment:
shouldn't this rather be `if (arg_mode != ID128_PRINT_UUID) …`? simply because the only acceptable values here are uuid and id128 really?

Reply:
Functionally, both are equivalent. But I added `arg_mode = ID128_PRINT_PRETTY` above, so it seems better to be symmetric and check for this condition here too.